pycircleci/api.py

```diff
@@ -867,6 +902,30 @@ def project_slug(self, username, reponame, vcs_type=GITHUB):
         slug = "{0}/{1}/{2}".format(vcs_type, username, reponame)
         return slug
 
+    def owner_slug(self, username, vcs_type=GITHUB):
+        """Return owner slug as string
+
+        :param username: Org or user name.
+        :param vcs_type: VCS type (github, bitbucket). Defaults to ``github``.
+
+        Returns: string ``:vcs-type/:username"``
+        """
+
+        slug = "{0}/{1}".format(vcs_type, username)
+        return slug
+
+    def owner_slug_json(self, username, vcs_type=GITHUB):
```

Comment:
not sure if this really deserves its own function; could hard-code it where it's needed, but this probably is the best style since I think we'll need it for some other calls as well?

I tried to see if we could use the "slug in endpoint" pattern, but doesn't seem feasible for this endpoint.

Reply:
I'd go with something like this
```
def owner_slug(self, username, format=None, vcs_type=GITHUB):
  slug = "{0}/{1}".format(vcs_type, username)
  if format == "json":
     slug = json.dumps({"owner-slug": slug})
   return slug

```

Reply:
good idea. format is reserved, so maybe I'll use `output`?

Reply:
Ended up pulling it out because of adding the other parameter options.

Reply:
other alternatives would be an arg like: `fmt=None|json` or `json=None|True|False`